python/ray/_private/worker.py

```diff
@@ -1263,6 +1263,15 @@ def init(
         usage_lib.record_library_usage("client")
         return ctx
 
+    if kwargs.get("allow_multiple"):
+        raise RuntimeError(
+            "`allow_multiple` argument is passed to `ray.init` when the "
+            "ray client is not used ("
+            "https://docs.ray.io/en/master/cluster/running-applications/job-submission"
```

Comment:
Actually can you replace master with `{get_ray_doc_version()}` to make sure the link will always be valid?